src/pasta.rs

```diff
@@ -89,13 +89,13 @@ impl Pasta {
 
     pub fn total_size_as_string(&self) -> String {
         let total_size_bytes = if self.has_file() {
-            self.file.as_ref().unwrap().size.as_u64() as usize + self.content.as_bytes().len()
+            self.file.as_ref().unwrap().size.as_u64() as usize + self.content.len()
         } else {
-            self.content.as_bytes().len()
+            self.content.len()
         };
 
         if total_size_bytes < 1024 {
```

Comment:
we might wanna throw in a dependency for this, I think https://crates.io/crates/humansize is a great fit (again seperate PR has nothing to do with your work here just something I noticed)